sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/CbdeHandler.CSharp9.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
```

Comment:
also add the `init` setter, static lambdas, discard params, and some top level statements (and the test should be given `outputKind: OutputKind.ConsoleApplication`)